GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -1,14 +1,64 @@
+/**
+ * An client side implementation of the Debugger
+ * @interface
+ */
+gdjs.DebuggerClient = function() {} // Necessary for Interface definition 
+
+/**
+ * Update a value of the {@link RuntimeGame} instance.
+ * @function
+ * @name gdjs.DebuggerClient#set
+ * @param {?} path - The path to modify.
```

Comment:
Why is there a "?"? 
If it's because you're unsure, prefer to put any and a TODO :)

Reply:
? is also a valid JSDoc type and is the same as any. I put it here to get your attention so you could correct me if it is wrong 😅

Reply:
Ok sure then :) 
path is an array of string.